katalog/tigera/README.md

```diff
@@ -0,0 +1,77 @@
+# Tigera Package
+
+The Tigera package provides the Tigera Operator and some ready to go configurations to enable Networking capabilities for a Kubernetes cluster.
+
+## Tigera Operator
+
+The Tigera Operator is an **alternative** to the [Calico](../calico) manifests package. It provides CNI and networking capabilities to a Kubernetes cluster.
+
+### On-premises installation
+
+To install the Tigera operator in an empty on-premises cluster run the following command:
+
+1. Deploy the `on-prem` package, it will deploy both the Operator and the configuration:
+
+```bash
+kustomize build katalog/tigera/on-prem | kubectl apply -f - --server-side
+```
+
+If you would like customize the installation, patch the `tigera/on-prem/custom-resources.yaml` your desired configuration. See the [official documentation](https://projectcalico.docs.tigera.io/getting-started/kubernetes/installation/config-options) for details.
+
+#### Migrating from Calico manifests to the Tigera Operator
+
+To migrate from a manifests installation to the Tigera operator the high-level steps are:
+
+1. Install the `on-prem` version:
+
+```bash
+kustomize build katalog/tigera/on-prem | kubectl apply -f - --server-side
+```
+
+2. The operator will adopt the existing resources and migrate them to the `calico-system` namespace.
+
+Please refer to Calico's documentation for more details:
+<https://projectcalico.docs.tigera.io/maintenance/operator-migration>
+
+> ⚠️ If you were using "**Infra**" nodes, you'll need to patch the `Install` resource with the right `NodeSelector` and `Tolerations`.
+> For example with the following Kustomize `patchesStrategicMerge`:
+>
+> ```yaml
+> ---
+> apiVersion: operator.tigera.io/v1
+> kind: Installation
+> metadata:
+>   name: default
```

Comment:
Is this CR namespaced?

Reply:
It is cluster-scoped. See line number 5464 of the CRD definition file `tigera-operator.yaml`